src/ProjectNeon/Assets/Cards/Effects/Actions/BuffAction.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu()]
+public class BuffAction : ScriptableObject

```

Comment:
How will we know what kind of buff to apply?

Will this increase Physical Damage amounts?
Will this increase Armor?
Will this let a character play an extra card?

We need some design that will support Polymorphism.

Reply:
@SilasReinagel I think that Buff cards are too complex to model this early; we barely have character attributes and enemy stats defined, so creating these effects so early in project would just make we have to refactor them later with no significant gains for making it now. I propose to evolve basic card and effects behavior and leave a puzzle for ceating buffs in the future